README.md

```diff
@@ -1,5 +1,9 @@
 # wtf-packages-report
 
+Get started via
+
+usethis::use_course("https://github.com/rstats-wtf/wtf-packages-report/archive/master.zip")
```

Comment:
````suggestion
```r
usethis::use_course("rstats-wtf/wtf-packages-report")
```
````

Reply:
If you format this as code it is more readable, gets syntax-highlighted, and gets an automatic copy/paste button.

I'm also using the much shorter form of `use_course()`.